Stripe/NSMutableURLRequest+Stripe.h

```diff
@@ -0,0 +1,15 @@
+//
```

Comment:
Make sure to load any categories like this in STPCategoryLoader to avoid messing up static builds & fabric.